src/com/company/model/CreateColumnDefinitionProvider.java

```diff
@@ -0,0 +1,27 @@
+package com.company.model;
+
+import com.company.view.ConsoleReader;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by yulia on 21.02.17.
+ */
+public class CreateColumnDefinitionProvider {
```

Comment:
Seems like this class provides properties, so it should be named **CreateColumnDefinitionPropertiesProvider**